hack/verify-publishing-bot.py

```diff
@@ -59,9 +59,15 @@ def main():
     components.sort()
 
     rules_file = "/staging/publishing/rules.yaml"
-    gomod_dependencies = get_gomod_dependencies(rootdir + '/staging/src/k8s.io/', components)
+    try:
+        import yaml
+    except ImportError:
+        print("Missing pyyaml module - Skipping verification of %s" % rules_file)
+        sys.exit(0)
```

Comment:
Do we really want to exit 0 if we can't run this verification? :\

Reply:
exit(1) would fail the verify which was my initial thought ... but went with (0) and printing the message so folks know that we are skipping something at least (and the rest of the verification can be done locally)

Reply:
I'd rather explicitly fail. It's still not a great local story, but at least you know this way that it's failing.